addons/website_slides/tests/test_slide_channel.py

```diff
@@ -138,18 +138,32 @@ def test_mail_completed_with_different_templates(self):
     @users('user_officer')
     def test_share_without_template(self):
         channel_without_template = self.env['slide.channel'].create({
```

Comment:
In case you have to amend for any other reason.
```suggestion
        channel_without_templates = self.env['slide.channel'].create({
```